ruledata.go

```diff
@@ -134,24 +133,32 @@ done:
 
 			}
 			if m.Header.Type == syscall.NLMSG_ERROR {
-				log.Println("NLMSG_ERROR\n")
+				error := int32(nativeEndian().Uint32(m.Data[0:4]))
+				if error == 0 {
+					log.Println("Acknowledged!!")
```

Comment:
Yes, I was already thinking of replacing the whole `log.Println` and `log.Fatalln` things as it will be difficult to differentiate between errors and logs. Errors and warning should be redirected on `stderr` and logs on `stdout`.
However `error := int32(nativeEndian().Uint32(m.Data[0:4]))` is quite necessary to the code to check for ACK.
I'll just create an issue to keep us reminded to discuss on this.